Fix JavaDoc

Currently:
```
$ mvn javadoc:javadoc
[INFO] Scanning for projects...
[INFO] 
[INFO] ----------------< net.imglib2:imglib2-label-multisets >-----------------
[INFO] Building ImgLib2 Label Multisets 0.9.1-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] >>> maven-javadoc-plugin:3.1.0:javadoc (default-cli) > generate-sources @ imglib2-label-multisets >>>
[INFO] 
[INFO] --- maven-enforcer-plugin:1.4.1:enforce (enforce-rules) @ imglib2-label-multisets ---
[INFO] Adding ignore: module-info
[INFO] Adding ignore: META-INF/versions/*/module-info
[INFO] Adding ignore: org.apache.hadoop.yarn.*.package-info
[INFO] Adding ignore: org.apache.spark.unused.UnusedStubClass
[INFO] Adding ignore: org.hibernate.stat.ConcurrentStatisticsImpl
[INFO] Adding ignore: org.junit.runner.Runner
[WARNING] Could not find junit:junit:jar:4.12:test at null
[INFO] Adding ignore: module-info
[INFO] 
[INFO] --- build-helper-maven-plugin:3.0.0:regex-property (sanitize-version) @ imglib2-label-multisets ---
[INFO] 
[INFO] --- buildnumber-maven-plugin:1.4:create (default) @ imglib2-label-multisets ---
[INFO] Executing: /bin/sh -c cd '/home/hanslovskyp/workspace/paintera/imglib2-label-multisets' && 'git' 'rev-parse' '--verify' 'HEAD'
[INFO] Working directory: /home/hanslovskyp/workspace/paintera/imglib2-label-multisets
[INFO] Storing buildNumber: a7f029c9a6841e62bbc55a48de06f980bcc0a88d at timestamp: 1570145683222
[INFO] Storing buildScmBranch: master
[INFO] 
[INFO] --- scijava-maven-plugin:1.1.0:set-rootdir (set-rootdir) @ imglib2-label-multisets ---
[INFO] Setting rootdir: /home/hanslovskyp/workspace/paintera/imglib2-label-multisets
[INFO] 
[INFO] --- jacoco-maven-plugin:0.8.4:prepare-agent (jacoco-initialize) @ imglib2-label-multisets ---
[INFO] argLine set to -javaagent:/home/hanslovskyp/.m2/repository/org/jacoco/org.jacoco.agent/0.8.4/org.jacoco.agent-0.8.4-runtime.jar=destfile=/home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/jacoco.exec
[INFO] 
[INFO] <<< maven-javadoc-plugin:3.1.0:javadoc (default-cli) < generate-sources @ imglib2-label-multisets <<<
[INFO] 
[INFO] 
[INFO] --- maven-javadoc-plugin:3.1.0:javadoc (default-cli) @ imglib2-label-multisets ---
[INFO] 
Loading source files for package net.imglib2.type.label...
Constructing Javadoc information...
Standard Doclet version 1.8.0_222
Building tree for all the packages and classes...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/AbstractLabelMultisetLoader.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/ByteUtils.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/FromIntegerTypeConverter.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/Label.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/Label.ReservedValue.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/LabelMultiset.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/LabelMultisetEntry.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/LabelMultisetEntryList.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/LabelMultisetToUnsignedLongConverter.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/LabelMultisetType.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/LabelMultisetType.Entry.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/LabelMultisetTypeDownscaler.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/LabelUtils.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/LongMappedAccess.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/LongMappedAccessData.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/MappedAccess.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/MappedAccessData.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/MappedAccessData.Factory.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/MappedObject.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/MappedObjectArrayList.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/Multiset.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/RefList.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/RefList.RefIterator.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/Test.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/VolatileLabelMultisetArray.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/VolatileLabelMultisetType.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/package-frame.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/package-summary.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/net/imglib2/type/label/package-tree.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/constant-values.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/serialized-form.html...
Building index for all the packages and classes...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/overview-tree.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/index-all.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/deprecated-list.html...
Building index for all classes...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/allclasses-frame.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/allclasses-noframe.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/index.html...
Generating /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs/help-doc.html...
12 errors
19 warnings
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  49.500 s
[INFO] Finished at: 2019-10-03T19:35:28-04:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:3.1.0:javadoc (default-cli) on project imglib2-label-multisets: An error has occurred in Javadoc report generation: 
[ERROR] Exit code: 1 - javadoc: warning - Error fetching URL: https://javadoc.scijava.org/ImageJ1/
[ERROR] javadoc: warning - Error fetching URL: https://javadoc.scijava.org/Eclipse/
[ERROR] javadoc: warning - Error fetching URL: https://javadoc.scijava.org/JOML/
[ERROR] javadoc: warning - Error fetching URL: https://javadoc.scijava.org/Javassist/
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/ByteUtils.java:14: error: malformed HTML
[ERROR]  * @author Tobias Pietzsch <tobias.pietzsch@gmail.com>
[ERROR]                            ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/ByteUtils.java:14: error: bad use of '>'
[ERROR]  * @author Tobias Pietzsch <tobias.pietzsch@gmail.com>
[ERROR]                                                      ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/Label.java:68: warning: no description for @param
[ERROR] 	 * @param a
[ERROR] 	   ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/Label.java:69: warning: no description for @param
[ERROR] 	 * @param b
[ERROR] 	   ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/Label.java:70: warning: no description for @return
[ERROR] 	 * @return
[ERROR] 	   ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/LabelMultiset.java:72: warning: no @param for data
[ERROR] 	protected void referToDataAt( final LongMappedAccessData data, final long baseOffset )
[ERROR] 	               ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/LabelMultiset.java:72: warning: no @param for baseOffset
[ERROR] 	protected void referToDataAt( final LongMappedAccessData data, final long baseOffset )
[ERROR] 	               ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/LabelMultisetEntryList.java:153: warning: no description for @param
[ERROR] 	 * @param list
[ERROR] 	   ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/MappedObjectArrayList.java:51: warning: no @param for data
[ERROR] 	protected void referToDataAt( final MappedAccessData< T > data, final long baseOffset )
[ERROR] 	               ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/MappedObjectArrayList.java:51: warning: no @param for baseOffset
[ERROR] 	protected void referToDataAt( final MappedAccessData< T > data, final long baseOffset )
[ERROR] 	               ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/MappedAccessData.java:30: warning: no @param for access
[ERROR] 	public void updateAccess( final T access, final long baseOffset );
[ERROR] 	            ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/MappedAccessData.java:30: warning: no @param for baseOffset
[ERROR] 	public void updateAccess( final T access, final long baseOffset );
[ERROR] 	            ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/MappedAccessData.java:42: warning: no @param for size
[ERROR] 	public void resize( final long size );
[ERROR] 	            ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/LongMappedAccessData.java:5: error: reference not found
[ERROR]  * A {@link MappedElementArray} that stores {@link DoubleMappedElement
[ERROR]             ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/LongMappedAccessData.java:5: error: reference not found
[ERROR]  * A {@link MappedElementArray} that stores {@link DoubleMappedElement
[ERROR]                                                    ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/LongMappedAccessData.java:8: error: malformed HTML
[ERROR]  * @author Tobias Pietzsch <tobias.pietzsch@gmail.com>
[ERROR]                            ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/LongMappedAccessData.java:8: error: bad use of '>'
[ERROR]  * @author Tobias Pietzsch <tobias.pietzsch@gmail.com>
[ERROR]                                                      ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/LongMappedAccessData.java:14: error: reference not found
[ERROR] 	 * {@link #resize(int) resized}.
[ERROR] 	          ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/LongMappedAccessData.java:16: warning - Tag @link: can't find resize(int) in net.imglib2.type.label.LongMappedAccessData
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/MappedAccess.java:5: error: reference not found
[ERROR]  * {@link MappedElementArray#updateAccess(MappedAccess, long)}, the base offset
[ERROR]           ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/MappedAccess.java:18: error: reference not found
[ERROR]  * {@link MappedElementArray#updateAccess(MappedAccess, int)} needs to be in
[ERROR]           ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/MappedAccess.java:19: error: reference not found
[ERROR]  * the {@link MappedElementArray}, not here. This is because data might be split
[ERROR]               ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/MappedAccess.java:20: error: reference not found
[ERROR]  * up into several {@link MappedElementArray MappedElementArrays}, in which case
[ERROR]                           ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/MappedAccessData.java:10: error: @param name not found
[ERROR]  * @param <A>
[ERROR]            ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/MappedAccessData.java:56: warning: no @param for size
[ERROR] 		public A createStorage( final long size );
[ERROR] 		         ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/MappedAccessData.java:56: warning: no @return
[ERROR] 		public A createStorage( final long size );
[ERROR] 		         ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/MappedAccessData.java:61: warning: no @return
[ERROR] 		public T createAccess();
[ERROR] 		         ^
[ERROR] 
[ERROR] Command line was: /usr/lib/jvm/java-8-openjdk/jre/../bin/javadoc -J-Xmx1024m @options @packages
[ERROR] 
[ERROR] Refer to the generated Javadoc files in '/home/hanslovskyp/workspace/paintera/imglib2-label-multisets/target/site/apidocs' dir.
[ERROR] 
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

```

```
[ERROR] Exit code: 1 - javadoc: warning - Error fetching URL: https://javadoc.scijava.org/ImageJ1/
[ERROR] javadoc: warning - Error fetching URL: https://javadoc.scijava.org/Eclipse/
[ERROR] javadoc: warning - Error fetching URL: https://javadoc.scijava.org/JOML/
[ERROR] javadoc: warning - Error fetching URL: https://javadoc.scijava.org/Javassist/
```

Note that the above are just `warning`s.

The blocking errors are these:

```
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/ByteUtils.java:14: error: malformed HTML
[ERROR]  * @author Tobias Pietzsch <tobias.pietzsch@gmail.com>
[ERROR]                            ^
[ERROR] /home/hanslovskyp/workspace/paintera/imglib2-label-multisets/src/main/java/net/imglib2/type/label/ByteUtils.java:14: error: bad use of '>'
[ERROR]  * @author Tobias Pietzsch <tobias.pietzsch@gmail.com>
[ERROR] 
```

--- a/src/main/java/net/imglib2/type/label/ByteUtils.java
+++ b/src/main/java/net/imglib2/type/label/ByteUtils.java
@@ -11,7 +11,7 @@ import sun.misc.Unsafe;
  * ({@code long, double} etc.) from bytes at an offset in a {@code long[]}
  * array.
  *
- * @author Tobias Pietzsch <tobias.pietzsch@gmail.com>
+ * @author Tobias Pietzsch &lt;tobias.pietzsch@gmail.com&gt;
  */
 //@SuppressWarnings( "restriction" )
 public class ByteUtils

--- a/src/main/java/net/imglib2/type/label/LongMappedAccessData.java
+++ b/src/main/java/net/imglib2/type/label/LongMappedAccessData.java
@@ -2,17 +2,18 @@ package net.imglib2.type.label;
 
 
 /**
- * A {@link MappedElementArray} that stores {@link DoubleMappedElement
- * DoubleMappedElements} in a {@code double[]} array.
+ * A {@link MappedAccessData} that stores {@link LongMappedAccess
+ * LongMappedElements} in a {@code long[]} array.
  *
- * @author Tobias Pietzsch <tobias.pietzsch@gmail.com>
+ * @author Tobias Pietzsch  &lt;tobias.pietzsch@gmail.com&gt;
  */
 public class LongMappedAccessData implements MappedAccessData< LongMappedAccess >
 {
-	/**
-	 * The current data storage. This is changed when the array is
-	 * {@link #resize(int) resized}.
-	 */
+
+  /**
+   * The current data storage. This is changed when the array is
+   * {@link #resize(long) resized}.
+   */
 	protected long[] data;
 
 	private long size;

--- a/src/main/java/net/imglib2/type/label/MappedAccess.java
+++ b/src/main/java/net/imglib2/type/label/MappedAccess.java
@@ -2,7 +2,7 @@ package net.imglib2.type.label;
 
 /**
  * Maps into region of underlying memory area (a primitive array or similar). By
- * {@link MappedElementArray#updateAccess(MappedAccess, long)}, the base offset
+ * {@link MappedAccessData#updateAccess(MappedAccess, long)}, the base offset
  * of this {@link MappedAccess} in the memory area can be set. Values of
  * different types can be read or written at (byte) offsets relative to the
  * current base offset. For example {@code putLong( 42l, 2 )} would put write
@@ -15,9 +15,9 @@ package net.imglib2.type.label;
  *
  * <p>
  * Note: The method for updating the base offset
- * {@link MappedElementArray#updateAccess(MappedAccess, int)} needs to be in
- * the {@link MappedElementArray}, not here. This is because data might be split
- * up into several {@link MappedElementArray MappedElementArrays}, in which case
+ * {@link MappedAccessData#updateAccess(MappedAccess, long)}} needs to be in
+ * the {@link MappedAccessData}, not here. This is because data might be split
+ * up into several {@link MappedAccessData MappedElementArrays}, in which case
  * the reference to the memory area must be updated in addition to the base
  * offset.
  *

--- a/src/main/java/net/imglib2/type/label/MappedAccessData.java
+++ b/src/main/java/net/imglib2/type/label/MappedAccessData.java
@@ -7,8 +7,6 @@ package net.imglib2.type.label;
  * grow, see {@link #resize(long)}, which involves reallocating and copying the
  * underlying primitive array.
  *
- * @param <A>
- *            recursive type of this container type.
  * @param <T>
  *            the {@link MappedAccess} used to access the container.
  *

